app/client/__tests__/components/identity/MarketingToggle.tsx

```diff
@@ -0,0 +1,53 @@
+import { cleanup, fireEvent, render } from "@testing-library/react";
+import React from "react";
+import { create } from "react-test-renderer";
+import { MarketingToggle } from "../../../components/identity/MarketingToggle";
```

Comment:
I think it would be best to stick with `@testing-library/react` instead of also using `react-test-renderer`, particularly if using the latter  with snapshot testing. Partially because it is a little confusing to mix approaches like this, but also because I have found that testing-library encourages you to write tests that are more legible in their purpose, and more resilient to future changes / refactoring.

You can use testing-library to implement snapshot testing as well, but I think it is better to write your assertions using queries. Makes it clearer which behaviours you actually care about, whereas snapshots can include things like auto-generated classnames.

Reply:
Snapshot tests have been removed in favour of queries for this and the MarketingCheckbox